plugins/community/nipunn1313-mypy-grpc/v3.6.0/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mypy-protobuf==3.5.0
+protobuf==4.23.4
```

Comment:
If you install just `mypy-protobuf` in a container and then run `pip freeze`, you'll get the full contents of this file based on https://github.com/nipunn1313/mypy-protobuf/blob/745a1977fc06e1e9eb59afbc4b90e4abf1f4858d/setup.cfg#L17-L19.